services/net/indexing/TNO.Services.Indexing.csproj

```diff
@@ -21,7 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Elastic.Clients.Elasticsearch" Version="8.0.0-beta.4" />
+    <PackageReference Include="Elastic.Clients.Elasticsearch" Version="8.0.6" />
```

Comment:
This library supports System.Text.Json configuration.  However, I am not able to figure out how to do low level requests, or date range queries.